var/spack/repos/builtin/packages/git/package.py

```diff
@@ -137,25 +138,26 @@ class Git(AutotoolsPackage):
             description='Enable native language support')
     variant('man', default=True,
             description='Install manual pages')
+    variant('subtree', default=True,
+            description='Add git-subtree command and capability')
 
+    depends_on('autoconf', type='build')
+    depends_on('automake', type='build')
     depends_on('curl')
     depends_on('expat')
     depends_on('gettext', when='+nls')
     depends_on('iconv')
     depends_on('libidn2')
+    depends_on('libtool',  type='build')
+    depends_on('m4',       type='build')
     depends_on('openssl')
     depends_on('pcre', when='@:2.13')
     depends_on('pcre2', when='@2.14:')
     depends_on('perl', when='+perl')
     depends_on('zlib')
     depends_on('openssh', type='run')
-
-    depends_on('autoconf', type='build')
-    depends_on('automake', type='build')
-    depends_on('libtool',  type='build')
-    depends_on('m4',       type='build')
```

Comment:
Can we keep build dependencies together? In general, I think it improves readability of packages

Reply:
Sure, I don't have a strong opinion either way.  I just pushed changes that put these deps together @alalazo